runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -109,10 +109,10 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 
 	void rememberObjectToRescan(MM_EnvironmentBase *env, J9Object *object);
 
-	MMINLINE bool isSATBBarrierActive(MM_EnvironmentBase* env)
+	MMINLINE bool isSATBBarrierActive()
 	{
 		return ((_extensions->configuration->isSnapshotAtTheBeginningBarrierEnabled()) &&
-				(!_extensions->sATBBarrierRememberedSet->isGlobalFragmentIndexPreserved(env)));
+				(!_extensions->sATBBarrierRememberedSet->isGlobalFragmentIndexPreserved()));
```

Comment:
Curious if the first condition was needed, at all!?

Reply:
it's needed, because `sATBBarrierRememberedSet` will be NULL otherwise. So we must pass that to make sure we're checking valid `sATBBarrierRememberedSet`... it can be replaced by a NULL check instead

Reply:
in theory, we could instantiate sATBBarrierRememberedSet always, but not sure if it's worth it (probably this method is not that hot)